docker/build_nginx.sh

```diff
@@ -52,15 +53,21 @@ rm v${RTMP_MODULE_VERSION}.tar.gz
 
 cd /tmp/nginx
 
+ARCH=$(uname -m)
+CC_OPT="-O3 -Wno-error=implicit-fallthrough"
+
+if [ "$ARCH" = "aarch64" ]; then
+    CC_OPT="${CC_OPT} -march=armv8-a+crc"
+fi
+
 ./configure --prefix=/usr/local/nginx \
-    --with-file-aio \
```

Comment:
When testing with this branch I had to add this option back for frigate to work.